components/nds-filters/src/FilterSummary.js

```diff
@@ -0,0 +1,186 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import { Tag } from "@nice-digital/nds-tag";
+import RemoveIcon from "@nice-digital/icons/lib/Remove";
+import "./../scss/filter-summary.scss";
+
+export function FilterSummary({
+	children,
+	className,
+	sorting = [],
+	activeFilters = [],
+	headingLevel = 2,
+	...rest
+}) {
+	const HeadingLevel = "h" + headingLevel;
+
+	return (
+		<div className={classnames("filter-summary", className)} {...rest}>
+			<div className="filter-summary__count">
+				<HeadingLevel className="h5 mv--0">{children}</HeadingLevel>
+			</div>
+			{sorting.length > 0 && (
+				<ResultsSorting
+					inactive={sorting.filter(item => !item.active)}
+					active={sorting.filter(item => item.active)[0]}
+				/>
+			)}
+			{activeFilters.length > 0 && <ResultsFilters filters={activeFilters} />}
+		</div>
+	);
+}
+
+const populateMethodProperty = (onClick, method, elementType) => {
+	let defaultMethod = method || (elementType === "a" && "href") || "to";
+	if (onClick) return "onClick";
+	if (elementType === undefined) defaultMethod = "noelementtypeprovided";
+	return defaultMethod;
+};
+
+const defineElementType = (onClick, elementType) => {
+	let element = "a";
+	if (onClick) element = "button";
+	if (elementType) element = elementType;
+	return element;
+};
+
+const validateElementProps = element => {
+	if (element.type.displayName == "Link" && !element.props.to) {
+		return "Link component missing 'TO' attribute --->";
+	}
+};
+
+function ResultsFilters({ filters }) {
+	if (!filters) return null;
+	return (
+		<ul
+			className="filter-summary__filters hide-print"
+			aria-label="Active filters"
+		>
+			{filters.map(
+				({
+					label,
+					destination,
+					method,
+					elementType,
+					onClick,
+					className = ""
+				}) => {
+					const ElementType = defineElementType(onClick, elementType);
+					const props = {
+						className: classnames(["tag__remove", className]),
+						"aria-label": `Sort by ${label}`,
```

Comment:
This should say Remove not Sort by @johndavey72 can you fix pls?
```suggestion
						"aria-label": `Remove ${label} filter`,
```